desktop/apps/auction/templates/my_active_bids.jade

```diff
@@ -23,7 +23,7 @@ if myActiveBids && myActiveBids.length
         .auction-my-active-bids__bids-num (#{bidCount} Bid#{bidCount > 1 ? 's' : ''})
         if bid.sale_artwork.sale.is_live_open
           a.avant-garde-button-white.auction-my-active-bids__bid-live-button(
-              href=ViewHelpers.liveAuctionUrl(bid.sale_artwork.sale_id)
+              href=helpers.auction.liveAuctionUrl(bid.sale_artwork.sale_id, {user: true})
```

Comment:
I wouldn't switch types between calls unless the name is updated as well -- for consistency can you pass in `me` here? or maybe update the var name to be `isLoggedIn`? Then you'd be able to do `isLoggedIn: Boolean(me)`